packages/platform-ios/native_modules.rb

```diff
@@ -87,7 +88,11 @@ def use_native_modules!(config = nil)
 
     relative_path = podspec_dir_path.relative_path_from project_root
 
-    pod spec.name, :path => relative_path.to_path
+    if buildTypes.nil? || buildTypes.empty?
```

Comment:
Since this is Ruby, one could wonder if this can be simplified. 

(would something happen if the code was just: `pod spec.name, :path => relative_path.to_path, :configurations => buildTypes`)?